src/pages/products/index.tsx

```diff
@@ -0,0 +1,33 @@
+import type { NextPage } from "next";
+import styled from "@emotion/styled";
+import ProductsList from "../../components/ProductsList";
+import Product from "../../components/Product";
+
+const ProductPage: NextPage = () => {
```

Comment:
p4: 개인적인 의견인데,  style이 너무 많아지면 아예 이렇게 함수 앞 쪽에 써주는 것을 저는 선호합니다. 참고만 해주세요!
앞에 써 놓으면 어떤 것을 export 하는지 바로 보여서 좋더라구요
```suggestion
export default const ProductPage: NextPage = () => {
```

Reply:
넵 이부분은 함께 얘기 나눠서 정해봐도 좋을 것 같아요!!